app/cli/internal/policydevel/eval.go

```diff
@@ -0,0 +1,162 @@
+// Copyright 2025 The Chainloop Authors.
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+package policydevel
+
+import (
+	"context"
+	"fmt"
+	"os"
+
+	v1 "github.com/chainloop-dev/chainloop/app/controlplane/api/workflowcontract/v1"
+	"github.com/chainloop-dev/chainloop/pkg/casclient"
+	"github.com/chainloop-dev/chainloop/pkg/policies"
+	"github.com/rs/zerolog"
+
+	v12 "github.com/chainloop-dev/chainloop/pkg/attestation/crafter/api/attestation/v1"
+	"github.com/chainloop-dev/chainloop/pkg/attestation/crafter/materials"
+)
+
+type EvalOptions struct {
+	PolicyPath   string
+	MaterialKind string
+	Annotations  map[string]string
+	MaterialPath string
+	Inputs       map[string]string
+}
+
+type EvalResult struct {
+	NoPolicies  bool
+	Skipped     bool
+	SkipReasons []string
+	Violations  []string
+}
+
+func Evaluate(opts *EvalOptions, logger zerolog.Logger) (*EvalResult, error) {
+	// 1. Create crafting schema
+	schema, err := createCraftingSchema(opts.PolicyPath, opts.Inputs)
+	if err != nil {
+		return nil, fmt.Errorf("creating crafting schema: %w", err)
+	}
+
+	// 2. Craft material with annotations
+	material, err := craftMaterial(opts.MaterialPath, opts.MaterialKind, &logger)
+	if err != nil {
+		return nil, fmt.Errorf("material crafting: %w", err)
+	}
+	material.Annotations = opts.Annotations
+
+	// 3. Verify material against policy
+	result, err := verifyMaterial(schema, material, opts.MaterialPath, &logger)
+	if err != nil {
+		return nil, err
+	}
+
+	return result, nil
+}
+
+func createCraftingSchema(policyPath string, inputs map[string]string) (*v1.CraftingSchema, error) {
+	return &v1.CraftingSchema{
+		Policies: &v1.Policies{
+			Materials: []*v1.PolicyAttachment{
+				{
+					Policy: &v1.PolicyAttachment_Ref{Ref: fmt.Sprintf("file://%s", policyPath)},
+					With:   inputs,
+				},
+			},
+			Attestation: nil,
+		},
+		PolicyGroups: nil,
+	}, nil
+}
+
+func verifyMaterial(schema *v1.CraftingSchema, material *v12.Attestation_Material, materialPath string, logger *zerolog.Logger) (*EvalResult, error) {
+	v := policies.NewPolicyVerifier(schema, nil, logger)
+	evs, err := v.VerifyMaterial(context.Background(), material, materialPath)
+	if err != nil {
+		return nil, err
+	}
+
+	result := &EvalResult{
+		NoPolicies:  true,
+		Skipped:     false,
+		SkipReasons: []string{},
+		Violations:  []string{},
+	}
```

Comment:
I think the `ignore` flag is missign

Reply:
Yes, @Piskoo there is a "ignore" flag in evaluation results, that is used for the engine to ignore the policy execution (as it never existed). It's used in cases where the decision logic is in the Rego itself. Not too important for this command though.